README.md

```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
 ### Setup Azure Account
 First, set up your Azure Account ([Get started for free!](https://azure.microsoft.com/en-us/free/))
```

Comment:
Can we also set up your Azure account through Azure Cloud Shell? At least there will be a prompt to login and it'll say create a new account? One less site to deal with?

Reply:
Yes, if they do not have an account yet, they will need to create it here.